app/controllers/conversations_controller.rb

```diff
@@ -21,21 +21,24 @@ def index
       end
     end
 
-    gon.contacts = contacts_data
-
     respond_with do |format|
-      format.html
+      format.html { render "index", locals: {no_contacts: current_user.contacts.empty?} }
       format.json { render json: @visibilities.map(&:conversation), status: 200 }
     end
   end
 
   def create
-    contact_ids = params[:contact_ids]
-
-    # Can't split nil
-    if contact_ids
-      contact_ids = contact_ids.split(',') if contact_ids.is_a? String
-      person_ids = current_user.contacts.where(id: contact_ids).pluck(:person_id)
+    contacts_ids = params["contacts-ids"]
```

Comment:
I think the parameter name should stay `contact_ids`, the double plural sounds weird.